### PR TITLE
Add SixthContinent

### PR DESCRIPTION
**Pull request by Nicos18**

### CODE REVIEW

**Review by stephengroat — DISMISSED** [review]

**Review by mxxcon — COMMENTED** [review]
Their official logo w/ background transparency.
sixthcontinent

Also I'm not sure if this should be in retail category or moved to finance?

**Comment by Nicos18** [comment]
This site lets you buy some virtual or physical cards with a discount, and receiving a cashback on every purchase. In addition, everyone receive some money from the users all over the world. Anyway, as the main purpose of this site is shopping, I think that the best category would be retail. Or, at last resort, maybe the other category

**Comment by mxxcon** [comment]
But this site is more of a e-wallet, rather than directly selling anything to customers.
Main purpose of credit cards is shopping too, but we don't put them in retail. 😉

**Comment by Nicos18** [comment]
So I should move this site to "finance" category?

**Comment by mxxcon** [comment]
@2factorauth/core comments on this?

**Comment by jamcat22** [comment]
Maybe it should be put into `payments` with Privacy?

**Comment by Carlgo11** [comment]
bump @2factorauth/collaborators

**Comment by brycx** [comment]
I agree with @jamcat22, since that's where most services with the same functionalities are listed. Though they seem to have a new logo and [Facebook](https://www.facebook.com/sixthcontinent.it/). And their Twitter is no more.

**Review by Carlgo11 — DISMISSED** [review]
Please move to `Payments` as suggested in https://github.com/2factorauth/twofactorauth/pull/2776#issuecomment-349779365

File: _data/payments.yml [hunk 1]
```diff
@@ -141,6 +141,13 @@ websites:
       software: Yes
       doc: https://ravelin.zendesk.com/hc/en-us/articles/115001071665-Two-Factor-Authentication
 
+    - name: SixthContinent
+      url: https://sixthcontinent.com/
+      img: sixthcontinent.png
+      tfa: No
+      twitter: sixthcontinent
+      facebook: sixthcontinentcom
```
Comment (Carlgo11) [inline _data/payments.yml]: Is it just me or does the Twitter and Facebook pages not exist? 🤔

Comment (Nicos18) [reply _data/payments.yml]: I think that those were the old ones, I just updated the twitter and Facebook handle with the new ones

**Review by brycx — CHANGES_REQUESTED** [review]
@Nicos18 Be so kind and update to their new logo.

**Review by brycx — APPROVED** [review]

**Review by mxxcon — APPROVED** [review]

**Pull request merged**